docs/changelog.md

```diff
@@ -4,6 +4,12 @@ tocMaxDepth: 2
 
 # Redocly CLI changelog
 
+## 1.0.0-beta.97 (2022-05-10)
+
+### Features
+
+- Added the `--public` option to the `push` command. With this option, you can upload OpenAPI definitions and make them publicly accessible.
+
 ## 1.0.0-beta.96 (2022-05-06)
 
 Technical release for changing the package name.
```

Comment:
```suggestion
Technical release for changing the package name from `@redocly/openapi-cli` to `@redocly/cli`.
```
I know this isn't related to your PR, but :smile:

Reply:
Yeah, going to update)